frontend/src/components/Header.module.scss

```
@@ -0,0 +1,74 @@
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: var(--size-3) var(--size-7);
+  border-bottom: var(--border-size-1) solid var(--color-border);
+
+  &__home {
```

```suggestion
.Navbar {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  padding: var(--size-3) var(--size-7);
  border-bottom: var(--border-size-1) solid var(--color-border);

  &Home {
```

Et du coup dans ton composant ça va etre ça pour utiliser la class `style.NavabarHome`